website/pages/docs/k8s/installation/multi-cluster/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
 page_title: Multi-Cluster Federation Overview
-sidebar_title: Overview
```

Comment:
We would like for this to stay as "Overview" in the sidebar

Reply:
It will say that. The default behavior is for `index.mdx` pages within folders to be displayed as "Overview" in the sidebar automatically, and for their sidebar titles to represent the name of the folder itself.

Reply:
Ahh yeah, sorry!

Reply:
This is noted under "a couple more important notes" in this section of the readme: https://github.com/hashicorp/consul/tree/master/website#editing-navigation-sidebars